src/gameplay/game.ts

```diff
@@ -21,6 +21,8 @@ class Game extends Room {
   phase = 'pickingTeam';
   missionHistory = [];
 
+  // Add const gameType = "Ranked"/"Unranked"/"Custom";
```

Comment:
@FakhruddinNalawala this should already be a variable. 

https://github.com/vck3000/ProAvalon/blob/master/src/gameplay/game.ts#L39

Reply:
Yeah, however, ranked seems insufficient for tracking the game type as it can only do ranked or unranked.
As for gameMode, I'll have to look into it further, as I am having difficulties parsing what it is.